### DISCUSSION TITLE
I_Message_Service Beispiele

### DISCUSSION THREAD

**Discussion by TheodorSzymkowiak** [gematik/api-kim#77] [General]
Hallo ehemalige Kollegen,
finde ich irgendwo eine Beispielnachricht für die I_Message_Service Schnittstelle oder ist das nur in den Specs über Anforderungen beschrieben? Ich würde gerne sehen, wie die Requests und Responses für die Operation send_Message aussehen, bzw. wie eine Standard-KIM-Nachricht (ohne Anhang) aufgebaut ist? 
Welche Parameter/Attribute hat eine Standard-Nachricht? Hintergrund ist, dass ich einen passenden Ort suche, an dem ich z.B. die Fallnummer des Patienten in der Nachricht übergeben kann.
Beste Grüße und Danke

**Comment by gem-uku** [comment]
Hallo Theo,
im Verzeichnis "samples" haben wir ein paar wenige Beispiele abgelegt. Die dort liegende Datei SMIME-Profil.zip soll einen vollständigen Verarbeitungsablauf zeigen, da die Datei nicht kontinuierlich aktualisiert wird, fehlen ggfs. neue oder geänderte X-KIM Header. Allerdings solltest du die prinzipiellen Strukturen erkennen können.

**Reply by TheodorSzymkowiak** [reply]
Vielen Dank für die Antwort. Gibt es irgendwo eine Übersicht, welche X-KIM Header pro Nachrichtentyp möglich sind? Wie ich herausgefunden habe hat z.B. die eNachricht unter umständen weniger Header-Attribute als ein eArztbrief? Konkretes Beispiel ist, ob die eNachricht auch eine "Content-Disposition" haben kann/darf.

**Comment by gem-uku** [comment]
Hallo Theo,
welche X-KIM Header von unserer Seite (gematik/KIM) gefordert werden, ist in unseren Spezifikationen zu finden. Spezielle Elemente, wie z. B. die Dienstkennung oder auch Bestandteile des Bodys ergeben sich dann allerdings aus den Spezifikationen der jeweiligen Anwendung. Bei dem von dir angesprochenen eArztbrief ist das meines Wissens die kv.digital.
